static/app/components/sentryDocumentTitle.tsx

```diff
@@ -28,7 +28,11 @@ function SentryDocumentTitle({title, orgSlug, projectSlug, children}: Props) {
 
   const docTitle = getDocTitle();
 
-  return <DocumentTitle title={`${docTitle} - Sentry`}>{children}</DocumentTitle>;
+  return (
+    <DocumentTitle title={`${docTitle} - Sentry`}>
+      {children as React.ReactChild}
```

Comment:
why do you need to cast children now?

Reply:
We cannot unfortunately, I don't know exactly what the issue is here, but types become incompatible farther up if I do that. I can look closer